.changeset/silly-crews-learn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@mysten/sui.js": minor
```

Comment:
just realized I should've made this a `patch` instead of `minor` but oops - lesson learned for next time 😅